src/editors/dateEditor/dateEditor.js

```diff
@@ -194,17 +191,6 @@ export class DateEditor extends TextEditor {
       // datepicker, but don't fill the editor input
       this.$datePicker.gotoToday();
     }
-
-    this.datePickerStyle.display = 'block';
-    this.$datePicker.show();
-  }
-
-  /**
-   * Hide data picker.
-   */
-  hideDatepicker() {
```

Comment:
Can I ask you to update the TS definition file (https://github.com/handsontable/handsontable/blob/feature/issue-6636/handsontable.d.ts#L392)? I guess that removing the `hideDatepicker` method from there is enough.

Reply:
Thank you for pointing unnecessary definitions out. I also removed the `types` test for `hideDatepicker()`.